cmd/dex/config.go

```diff
@@ -141,6 +141,8 @@ type OAuth2 struct {
 	AlwaysShowLoginScreen bool `json:"alwaysShowLoginScreen"`
 	// This is the connector that can be used for password grant
 	PasswordConnector string `json:"passwordConnector"`
+	// This is the default connector that can be used for password grant
+	DefaultPasswordConnector string `json:"defaultPasswordConnector"`
```

Comment:
The validation is required that both PasswordConnector and DefaultPasswordConnector can be specified. See the `Validate` method in this file.